package.json

```diff
@@ -149,10 +152,11 @@
     "package-hash": "^1.1.0",
     "pkg-conf": "^2.0.0",
     "plur": "^2.0.0",
-    "power-assert-context-formatter": "^1.0.4",
-    "power-assert-renderer-assertion": "^1.0.1",
-    "power-assert-renderer-succinct": "^1.0.1",
+    "pretty-format": "github:vadimdemedes/pretty-format#colors",
```

Comment:
@vadimdemedes What's the intention here? Upstream those changes? We cannot depend on a git branch, so we'll either have to upstream or you need to publish your fork to npm.